content/docs/community/contribution-guide.md

```diff
@@ -0,0 +1,415 @@
+enableTableOfContents | A boolean flag (i.e., true or false) that tells the static site generator whether or not to generate a right-hand table of contents for the page. We recommend adding this option if your page has more than a few sections.
+isDraft | Adding this attribute and setting it to `true` prevents the page from becoming a documentation page and being searchable. Include this option to avoid publishing the content before its ready.
+redirectFrom | A list of directory paths that should redirect to this file. This is useful if the page has moved and you want old URLs to continue working.
+updatedOn | This attribute is added automatically. You do not need to add it.
+
+Example:
+
+```yaml
+---
+title: Connect a Next.js application to Neon
+subtitle: Set up a Neon project and connect from a Next.js application
+enableTableOfContents: true
+redirectFrom:
+  - /docs/content/<old_directory_name>
+`updatedOn: '2023-10-07T12:25:27.662Z'`
+---
+```
+
+## Heading levels
+
+The page title in the frontmatter is translated into an `h1` element when the page is converted to HTML.
+
+Top-level section headings in the body of your document are defined with two hash characters, which is equivalent to an `h2` heading:
+
+```md
+## Section heading
+```
+
+To add a subsection heading, equivalent to an `h3`, add another `#` character:
+
+```md
+### Subsection heading
+```
+
+- Try to avoid heading levels beyond h4 (`####`).
+- Do not skip a level, e.g., do not go from `##` to `####`.
+- Ensure there's a blank line before and after each heading.
+
+## Common markup
+
+```md
+External link markup: [Example.com website](https://www.example.com/)
+Neon documentation page link: [Connection from any application](/docs/connect/connect-from-any-app)
+Neon documentation same page link: [Code blocks](#code-blocks)
+Italics markup: _italic_
+Bold markup: **strong**
+monospace: `backtick`
+```
+
+## Code blocks
+
+To insert a code block into your Markdown file, specify three backticks (```) on the lines before and after the code. Specify the language identifier to enable code highlighting, as in this example:
+
+````md
+```sql
+SELECT * FROM posts ORDER BY id;
+```
+````
+
+You can add language-specific highlighting to code blocks, as in the example above. See [Supported language highlighting for code blocks](https://prismjs.com/index.html#supported-languages).
+
+You can also wrap code blocks in `<CodeBlock></CodeBlock>` tags to show line numbers or wrap long strings.
+
+`<CodeBlock>` accepts the following options:
+
+- `showLineNumbers`: Shows the line numbers in the code block
+- `shouldWrap`: Enables code wrapping in the code block. This is useful for long commands or connection strings that would otherwise require the reader to scroll.
+
+Example:
+
+````md
+<CodeBlock shouldWrap>
+
+```text
+DATABASE_URL=postgres://alex:AbC123dEf@ep-cool-darkness-123456.us-east-2.aws.neon.tech/dbname?connect_timeout=15&pool_timeout=15`
+```
+
+</CodeBlock>
+````
+
```

Comment:
Does shouldWrap not work in preview? It's not wrapping for me on my browser (mac chrome):
![image](https://github.com/neondatabase/website/assets/5885546/d6558541-0325-4b47-9ea3-c92c63486c27)

Reply:
This example shows what the markdown looks like with the tags, so the line would not be wrapped here, but I've replaced the line to avoid any confusion.